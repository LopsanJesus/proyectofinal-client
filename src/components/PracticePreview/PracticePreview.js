import React from 'react';
import './PracticePreview.scss';

const PracticePreview = () => {
  return <div>PracticePreview</div>;
};

export default PracticePreview;
