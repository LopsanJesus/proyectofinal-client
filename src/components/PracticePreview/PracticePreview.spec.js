import React from 'react';
import { shallow } from 'enzyme';
import PracticePreview from './PracticePreview';

describe('PracticePreview', () => {
  test('matches snapshot', () => {
    const wrapper = shallow(<PracticePreview />);
    expect(wrapper).toMatchSnapshot();
  });
});
