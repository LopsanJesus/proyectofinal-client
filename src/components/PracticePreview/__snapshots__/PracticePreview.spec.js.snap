// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PracticePreview matches snapshot 1`] = `ShallowWrapper {}`;
